tests/cloudpickle_test.py

```diff
@@ -226,6 +226,24 @@ def test_dict_items(self):
         self.assertEqual(results, items)
         assert isinstance(results, _collections_abc.dict_items)
 
+    def test_odict_keys(self):
+        keys = collections.OrderedDict([("a", 1), ("b", 2)]).keys()
+        results = pickle_depickle(keys)
+        self.assertEqual(results, keys)
+        assert type(keys) == type(results)
+
+    def test_odict_values(self):
+        values = collections.OrderedDict([("a", 1), ("b", 2)]).values()
+        results = pickle_depickle(values)
+        self.assertEqual(sorted(results), sorted(values))
```

Comment:
Out of curiosity, can you explain why we to check for equality on the sorted values, and not the "raw" values?

Reply:
I think it's probably not necessary to sort. It is necessary to pull the individual elements out of the view. From [the documentation on dict.values()](https://docs.python.org/3/library/stdtypes.html#dict.values) (which also applies to OrderedDict.values()):
>>> An equality comparison between one dict.values() view and another will always return False.

I was basically copying the test for dict.values() which has sorted. Perhaps in older versions of Python and on a different implementation than CPython, regular dictionaries could potentially give values() in different order. OrderedDict should preserve the order for values() (though the documentation doesn't explicitly say it will) so I could change it from `sorted(...)` to `list(...)` to pull the values out if you'd like.

Reply:
@pierreglaser any preference on whether to change `sorted(...)` to `list(...)`?

Reply:
+1 for list.

Reply:
Done.